subgraphs/enzyme-core/mappings/v4/NoDepegOnRedeemSharesForSpecificAssetsPolicy.ts

```diff
@@ -0,0 +1,9 @@
+import { ensureNoDepegOnRedeemSharesForSpecificAssetsPolicy } from '../../entities/NoDepegOnRedeemSharesForSpecificAssetsPolicy';
+import { FundSettingsUpdated } from '../../generated/contracts/NoDepegOnRedeemSharesForSpecificAssetsPolicy4Events';
+
+export function handleFundSettingsSet(event: FundSettingsUpdated): void {
```

Comment:
The function name needs to be consistent with the event type, i.e. `handleFundSettingsUpdated`

Reply:
done